docs/scitt.md

```diff
@@ -0,0 +1,125 @@
+# SCITT Standard alignment
+
+# Signed Statement Inputs
+
+scitt-ccf-ledger accepts Signed Statement inputs as specified in Section 4.2 of the [Architecture Draft](https://datatracker.ietf.org/doc/draft-ietf-scitt-architecture/). See tests for details.
+
+# Registration Policy
+
+scitt-ccf-ledger implements registration policy as specified in Section 4.1.1 of the [Architecture Draft](https://datatracker.ietf.org/doc/draft-ietf-scitt-architecture/). See configuration for details. 
+
+# Hashed Envelope Format
+
+scitt-ccf-ledger has no specific support for the [Hashed Envelope Format Draft](https://cose-wg.github.io/draft-ietf-cose-hash-envelope/draft-ietf-cose-hash-envelope.html), which is not currently implementable because the Header Parameters it introduces have undefined value.
+
+Alignment will require that scitt-ccf-ledger passes these values through correctly, and exposes them to registration policy, which can be captured by a testcase.
+
+# Transparent Statement Output
+
+scitt-ccf-ledger returns Transparent Statement outputs as specified in Section 4.4 of the [Architecture Draft](https://datatracker.ietf.org/doc/draft-ietf-scitt-architecture/). See tests for details.
+
+# Receipts
+
+scitt-ccf-ledger returns COSE Receipts, either standalone or embedded in Transparent Statement outputs, as specified in Section 4 of the [COSE Receipt Draft](https://datatracker.ietf.org/doc/draft-ietf-cose-merkle-tree-proofs/), using proofs that follow the [CCF Tree algorithm profile Draft](https://datatracker.ietf.org/doc/draft-birkholz-cose-receipts-ccf-profile/).
+
+# API
+
+scitt-ccf-ledger does not currently implement the [SCITT Reference API](https://github.com/ietf-wg-scitt/draft-ietf-scitt-scrapi/blob/main/draft-ietf-scitt-scrapi.md). The following changes are necessary to align:
```

Comment:
FYI receipts endpoint is missing in the list

Reply:
There are only two mandatory endpoints in the current draft, the transparency configuration, and registration which can be async and return the location at which the receipt is eventually available. I think that's the scheme we are going for at the moment, but you're right that we can specify that too (I think it's a `GET application/cose`).

Reply:
Fair enough, let's not do that now as it is not mandatory